assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -330,15 +313,20 @@ void EntityTreeSendThread::startNewTraversal(const ViewFrustum& view, EntityTree
                                 _sendQueue.push(PrioritizedEntity(entity, PrioritizedEntity::WHEN_IN_DOUBT_PRIORITY));
                                 _entitiesInQueue.insert(entity.get());
                             }
+                        } else if (entity->getLastEdited() > knownTimestamp->second) {
+                            // it is known and it changed --> put it on the queue with any priority
+                            // TODO: sort these correctly
+                            _sendQueue.push(PrioritizedEntity(entity, PrioritizedEntity::WHEN_IN_DOUBT_PRIORITY));
+                            _entitiesInQueue.insert(entity.get());
```

Comment:
If:
- your view isn't moving
- an entity is in view and large enough
- it is edited to be too small, so it is added to the send queue here
- your view changes before it is sent
It will be removed from the send queue during the re-sort, so it will never update.

Reply:
Note: I moved the check of the _knownState timestamp from above down to here to avoid the extra work of checking for overlap and LOD:  if the entity is known and has changed then we should send an update.  At the moment the update is sent we could decide to remove from _knownState if it is no longer in view.

In short, I'm not adding a new problem here: the problem is fundamental to what the re-sort is doing.  We need to re-examine the re-sort's tendency to cull stuff out (maybe we should keep everything on the send-queue but just sort to-be-culled stuff to the end).